File: youlexuan-sellergoods_interface/src/main/java/com/offcn/sellergoods/service/BrandService.java
package com.offcn.sellergoods.service;

import com.offcn.pojo.PageResult;
import com.offcn.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {
    public List<TbBrand> findAll();

    public PageResult findPage(int pageNum, int pageSize);

    public void add(TbBrand tbBrand);

    public void update(TbBrand tbBrand);

    //根据id获取实体
    public TbBrand findOne(long id);
//删除
    public void delete(Long[] ids);
    //条件搜索
    public PageResult fingPage(TbBrand tbBrand,int pageNum,int pageSize);
//品牌下拉选择框
    public List<Map> selectOptionList();


}

File: youlexuan-manager-web/src/main/webapp/js/service/brandService.js
app.service("brandService", function ($http) {
    this.findAll = function () {
        return $http.get("../brand/findAll.do");
    }
    this.findPage = function (page, rows) {

        return $http.post("../brand/findPage.do?page=" + page + "&rows=" + rows);
    }
    this.save = function (name, entity) {
        return $http.post("../brand/" + name + ".do", entity);
    }
    this.findOne = function (id) {
        return $http.get("../brand/findOne.do?id=" + id);
    }
    this.dele = function (selectId) {
        return $http.get("../brand/delete.do?ids=" + selectId);
    }
    this.search = function (page, rows, tbBrand) {
        return $http.post("../brand/search.do?page=" + page + "&rows=" + rows, tbBrand);
    }
    this.selectOptionList=function () {

        return $http.get("../brand/selectOptionList.do")
    }
})